music/src/gansynth/audio_utils.ts

```diff
@@ -0,0 +1,243 @@
+/**
+ * ISTFT exactly matching python tensorflow.
+ *
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as tf from '@tensorflow/tfjs';
+//@ts-ignore
+import * as FFT from 'fft.js';
+
+import {applyWindow, frame, hannWindow} from './../transcription/audio_utils';
```

Comment:
please move audio_utils into core

Reply:
Maybe leave that for another time because some of these things are rather gansynth specific?